addon/webextension/selector/ui.js

```diff
@@ -610,9 +606,34 @@ this.ui = (function() { // eslint-disable-line no-unused-vars
       this.bgBottom = makeEl("div", "bghighlight");
       iframe.document().body.appendChild(this.bgBottom);
       iframe.document().body.appendChild(boxEl);
+      iframe.document().body.appendChild(this.buttons);
```

Comment:
Hmm. Might be a little weird that `this.buttons` is created in `_createButtons` but assumed to exist here. maybe `_createButtons` should return `buttons`, then this function can set `this.buttons = buttons` and append it to the DOM?